### PR TITLE
Add Terraform configs format check.

### PR DESCRIPTION
**Pull request by xingao267**
#10

### CODE REVIEW

**Comment by umairidris** [comment]
I am not too sure about this.. the configs themselves should be tested with go_test and fmt should be done as part of the engine binary.

The reason is there are some cases where we can't know what the final indentation etc will look like e.g. when we have optional vars being added.

Would having a go_test for all examples and running fmt on the generated configs automatically fix this?

File: build/presubmit.yaml [hunk 1]
```diff
@@ -16,10 +16,17 @@ steps:
 - name: "gcr.io/cloud-builders/npm"
   entrypoint: "sh"
   args: ["build/md_check.sh"]
+  waitFor: ["-"] # wait for nothing
```
Comment (MartinPetkov) [inline build/presubmit.yaml:19]: This is new to me, TIL.

**Comment by xingao267** [comment]
> I am not too sure about this.. the configs themselves should be tested with go_test and fmt should be done as part of the engine binary.
> 
> The reason is there are some cases where we can't know what the final indentation etc will look like e.g. when we have optional vars being added.
> 
> Would having a go_test for all examples and running fmt on the generated configs automatically fix this?
Yes. That will fix it. The test in the PR is just temporary so that before we have those implemented, and as we are actively make changes to the templates. I would like to have something very simple just to make sure they are also sanity checked. Because right now, everytime I make changes to the templates, I do that manually locally.

**Comment by umairidris** [comment]
Let me add this as a test after #21. It shouldn't take too long, just need to do a bit of minor refactoring.

**Comment by xingao267** [comment]
sgtm

**Pull request closed without merging**